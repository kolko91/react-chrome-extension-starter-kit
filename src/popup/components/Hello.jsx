import React from 'react';

export default class Hello extends React.Component {
  static propTypes = {}
  static defaultProps = {}
  state = { extends: false }
  render() {
    return (
      <div>
        Hello RCE Popup
      </div>
    );
  }
}
