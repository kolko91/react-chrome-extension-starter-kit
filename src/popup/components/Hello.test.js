import React from 'react';
import renderer from 'react-test-renderer';

import Hello from './Hello';

it('renders footer correctly', () => {
  const tree = renderer.create(<Hello />).toJSON();
  expect(tree).toMatchSnapshot();
});
