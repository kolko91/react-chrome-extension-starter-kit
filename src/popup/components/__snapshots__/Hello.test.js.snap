// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders footer correctly 1`] = `
<div>
  Hello RCE Popup
</div>
`;
